charts/dex/templates/deployment.yaml

```diff
@@ -96,7 +96,7 @@ spec:
       volumes:
         - name: config
           secret:
-            secretName: {{ include "dex.fullname" . }}
+            secretName: {{ if .Values.useExternalSecret.enabled }}{{ .Values.useExternalSecret.name }}{{ else }}{{ include "dex.fullname" . }}{{ end }}
```

Comment:
Can you please add a template to the end of the helpers file similar to the service account name template, based on the comments in the values file?